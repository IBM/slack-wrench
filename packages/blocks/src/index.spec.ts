import * as index from './index';

describe('The index', () => {
  it('exports everything', () => {
    expect.assertions(1);
    expect(Object.keys(index)).toMatchSnapshot();
  });
});
