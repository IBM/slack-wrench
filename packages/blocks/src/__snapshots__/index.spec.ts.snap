// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The index exports everything 1`] = `
Array [
  "Actions",
  "Context",
  "Divider",
  "Input",
  "Section",
  "MdSection",
  "FieldsSection",
  "Blocks",
  "Markdown",
  "PlainText",
  "OptionObject",
  "Button",
  "Image",
  "OverflowMenu",
  "PlainTextInputElement",
  "RadioInputElement",
  "StaticSelectInputElement",
  "DateString",
  "User",
]
`;
